packages/quantic/force-app/main/default/lwc/quanticResultText/quanticResultText.html

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="result-text slds-grid slds-grid_vertical-align-center">
```

Comment:
I'm not sure if the `result-text` CSS class is really needed here, I don't see it in the CSS file or in the tests.

Reply:
no honestly i thought i would use it then left it in as a potential hook point for a selector or something. I'll remove it though.